# Remove due to the snap crashing
diff --git a/launcher/ui/dialogs/MSALoginDialog.cpp b/launcher/ui/dialogs/MSALoginDialog.cpp
index be49babb..be20beae 100644
--- a/launcher/ui/dialogs/MSALoginDialog.cpp
+++ b/launcher/ui/dialogs/MSALoginDialog.cpp
@@ -48,7 +48,6 @@ MSALoginDialog::MSALoginDialog(QWidget *parent) : QDialog(parent), ui(new Ui::MS
 {
     ui->setupUi(this);
     ui->progressBar->setVisible(false);
-    ui->actionButton->setVisible(false);
     // ui->buttonBox->button(QDialogButtonBox::Cancel)->setEnabled(false);
 
     connect(ui->buttonBox, &QDialogButtonBox::accepted, this, &QDialog::accept);
@@ -105,17 +104,10 @@ void MSALoginDialog::showVerificationUriAndCode(const QUrl& uri, const QString&
     QString urlString = uri.toString();
     QString linkString = QString("<a href=\"%1\">%2</a>").arg(urlString, urlString);
     ui->label->setText(tr("<p>Please open up %1 in a browser and put in the code <b>%2</b> to proceed with login.</p>").arg(linkString, code));
-    ui->actionButton->setVisible(true);
-    connect(ui->actionButton, &QPushButton::clicked, [=]() {
-        DesktopServices::openUrl(uri);
-        QClipboard* cb = QApplication::clipboard();
-        cb->setText(code);
-    });
 }
 
 void MSALoginDialog::hideVerificationUriAndCode() {
     m_externalLoginTimer.stop();
-    ui->actionButton->setVisible(false);
 }
 
 void MSALoginDialog::setUserInputsEnabled(bool enable)
@@ -141,7 +133,6 @@ void MSALoginDialog::onTaskFailed(const QString &reason)
     // Re-enable user-interaction
     setUserInputsEnabled(true);
     ui->progressBar->setVisible(false);
-    ui->actionButton->setVisible(false);
 }
 
 void MSALoginDialog::onTaskSucceeded()
diff --git a/launcher/ui/dialogs/MSALoginDialog.ui b/launcher/ui/dialogs/MSALoginDialog.ui
index c18d01a1..c9eaba22 100644
--- a/launcher/ui/dialogs/MSALoginDialog.ui
+++ b/launcher/ui/dialogs/MSALoginDialog.ui
@@ -50,13 +50,6 @@ aaaaa</string>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
-     <item>
-      <widget class="QPushButton" name="actionButton">
-       <property name="text">
-        <string>Open page and copy code</string>
-       </property>
-      </widget>
-     </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">